github-mvi-android/app/src/main/java/com/rightpoint/github/mvi/repos/Intent.kt

```diff
@@ -0,0 +1,3 @@
+package com.rightpoint.github.mvi.repos
+
+data class Intent(val owner: String)
```

Comment:
#pp: I know the pattern is Model-View-Intent, but the naming here feels confusing if only because at quick glance one could confuse it with an Android intent.

Reply:
I'll probably rename it to `Action` because I feel that conveys what this class represents without the stated naming confusion.